woocommerce/woocommerce-android
# Issue/5456 simple payments taxes

Closes #5456 - this PR addresses the tax-related portion of simple payments. I recommend testing it with a store that has a tax rate, which can be configured under `WooCommerce > Settings > Tax > Standard Tax Rates`.

To test:

* Tap to create a simple payment
* Enter $100 as the amount and tap Done
* Verify the correct tax is shown on the Simple Payment screen
* Turn off "Charge taxes" and verify the tax no longer appears and is no longer included in the order total

Note that changes aren't saved, that will come in a later PR.

![sp](https://user-images.githubusercontent.com/3903757/146986215-8584d5bb-deb1-4994-8058-d44a921d01d2.png)
